test/AGENTS.md

```
@@ -1,11 +1,11 @@
 # TEST KNOWLEDGE BASE
 
-Generated: 2026-05-02
-Commit: b0ef65d
+Generated: 2026-07-29
+Commit: 78aa9b5 (2.8.0)
 
 ## OVERVIEW
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)